homeassistant/components/sensor/wink-eggminder.py

```diff
@@ -0,0 +1,63 @@
+"""
+homeassistant.components.egg_minder.wink
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for Wink Egg Minder.
+For more details about this platform, please refer to the documentation at
+at https://home-assistant.io/components/sensor.wink/
+"""
+import logging
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+REQUIREMENTS = ['https://github.com/balloob/python-wink/archive/'
+                '42fdcfa721b1bc583688e3592d8427f4c13ba6d9.zip'
```

Comment:
This needs update to the last version of python-wink 0.2.1 with sha `e66277ddb8ba51df34e810f784088c3c8582984a`. This has to happen in every other wink-related file too. After that, run `script/gen_requirements_all` to bring `requirements_all.txt` up to date.